### PR TITLE
Feint blade fix

### LINKED ISSUE #21214: Feint-given Cult Blade fucks up Backpack slot
(WEB REPORT BY: shadownil REMOTE: 206.221.180.138:7777)
# Revision
c79898e9dad35fe8bc5a6d0cbc7bef996f80d276

# Description
The spell Feint is described as: 
"Feint - cooldown: 30s (buy for 20 points)
This spell grants you a magic weapon and causes you to vanish, before reappearing behind the enemy."
The magic weapon in question is a cult blade, not sure if Cult 3 or Cult Legacy though. Anyway, I put the cult blade in my backpack, then checked it again. The cult blade had disappeared from my backpack storage slot, leaving an empty slot. I can fill up the backpack 7 slots, even though the 6th one is empty but static as if there's an item there when there is not.

# Steps to Reproduce
Be wiz
Buy Feint
Use it on someone
Put the blade in your backpack
Blade disappears
Put other shit in your backpack
6/7 slots inexplicably occupied

# What you Expected
all 7 slots to be visibly and physically filled with items

# What Actually Happened
The backpack is full, make some space.

I'm not sure if having the backpack storage window open or closed is relevant to this, but I put the Feint-given cult blade in my backpack without the inventory bar being open.

### PR DESCRIPTION
**Pull request by DeityLink**
Fixes #21214

image

:cl:
* imageadd: The wizard's Feint spell's enchanted sword finally has its own sprites, courtesy of Fury.
* bugfix: You can no longer bork your inventory slots by trying to store the enchanted blade.

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Looks wonderful.

**Comment by DeityLink** [comment]
there's also sprites for a projectile attack should we want to buff the spell.

File: code/game/objects/items/weapons/weaponry.dm [hunk 1]
```diff
@@ -148,16 +148,20 @@
 /obj/item/weapon/katana/magic
 	name = "enchanted sword"
```
Comment (jknpj) [inline code/game/objects/items/weapons/weaponry.dm]: requesting a name change to "moonlight-enchanted sword"

Comment (DeityLink) [reply code/game/objects/items/weapons/weaponry.dm]: can do

Comment (jknpj) [reply code/game/objects/items/weapons/weaponry.dm]: image

**Pull request merged**